components/nexusoperations/executors.go

```diff
@@ -298,6 +322,24 @@ func (e taskExecutor) saveResult(ctx context.Context, env hsm.Environment, ref h
 				return hsm.TransitionOutput{}, err
 			}
 			if result.Pending != nil {
+				var links []*commonpb.Link
+				for _, nexusLink := range result.Links {
+					switch {
+					case strings.HasPrefix(nexusLink.Type, string((&commonpb.Link_WorkflowEvent{}).ProtoReflect().Descriptor().FullName())):
+						link, err := ConvertNexusLinkToLinkWorkflowEvent(nexusLink)
+						if err != nil {
+							return hsm.TransitionOutput{}, err
+						}
```

Comment:
We should either ignore or fail the operation. Returning an error here would cause us to retry.

Reply:
Let's ignore for simplicity now.